src/main/java/org/zalando/jackson/datatype/money/MonetaryAmountSerializer.java

```diff
@@ -16,6 +16,7 @@
 
     private final MonetaryAmountFormatFactory factory;
     private final FieldNames names;
+    private final boolean numberAsString;
```

Comment:
Instead of using a boolean, can we use our own implementation of Jackson's Feature types?